Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/BloomDownsampleCS.azsl

```diff
@@ -12,12 +12,28 @@
 ShaderResourceGroup PassSrg : SRG_PerPass
 {
     Texture2D<float4>       m_sourceTexture;
+    [[vk::image_format("rgba16f")]]
```

Comment:
What happens if the the runtime binds a resource of different format? A crash or a validation errors?